website/content/api-docs/secret/pki.mdx

```diff
@@ -16,729 +16,735 @@ update your API calls accordingly.
+  to an issuer. This parameter is part of the request URL.
 
-## Parameters
+~> Note: This parameter is not present on the `/pki/root/sign-self-issued`
+   path and takes the value `default`.
 
-- `certificate` `(string: <required>)` – Specifies the certificate in PEM
-  format. May optionally append additional CA certificates to populate the
-  whole chain, which will then enable returning the full chain from issue and
-  sign operations.
+- `certificate` `(string: <required>)` - Specifies the PEM-encoded self-issued certificate.
+
+- `require_matching_certificate_algorithms` `(bool: false)` - If true, requires
+that the public key algorithm of the CA match that of the submitted certificate.
 
-### Sample Payload
+#### Sample Payload
 
 ```json
 {
-  "certificate": "-----BEGIN CERTIFICATE..."
+  "certificate": "..."
 }
 ```
 
-### Sample Request
+#### Sample Request
 
 ```shell-session
 $ curl \
     --header "X-Vault-Token: ..." \
     --request POST \
     --data @payload.json \
-    http://127.0.0.1:8200/v1/pki/intermediate/set-signed
+    http://127.0.0.1:8200/v1/pki/root/sign-self-issued
 ```
 
-## Generate Certificate
+#### Sample Response
 
-This endpoint generates a new set of credentials (private key and certificate)
-based on the role named in the endpoint. The issuing CA certificate is returned
-as well, so that only the root CA need be in a client's trust store.
+```json
+{
+  "lease_id": "",
+  "renewable": false,
+  "lease_duration": 0,
+  "data": {
+    "certificate": "-----BEGIN CERTIFICATE-----\nMIIDzDCCAragAwIBAgIUOd0ukLcjH43TfTHFG9qE0FtlMVgwCwYJKoZIhvcNAQEL\n...\numkqeYeO30g1uYvDuWLXVA==\n-----END CERTIFICATE-----\n",
+    "issuing_ca": "-----BEGIN CERTIFICATE-----\nMIIDUTCCAjmgAwIBAgIJAKM+z4MSfw2mMA0GCSqGSIb3DQEBCwUAMBsxGTAXBgNV\n...\nG/7g4koczXLoUM3OQXd5Aq2cs4SS1vODrYmgbioFsQ3eDHd1fg==\n-----END CERTIFICATE-----\n"
+  },
+  "auth": null
+}
+```
 
-**The private key is _not_ stored. If you do not save the private key, you will
-need to request a new certificate.**
+### Sign Verbatim
 
-| Method | Path               |
-| :----- | :----------------- |
-| `POST` | `/pki/issue/:name` |
+This endpoint signs a new certificate based upon the provided CSR. Values are
+taken verbatim from the CSR; the _only_ restriction is that this endpoint will
+refuse to issue an intermediate CA certificate (see the
+`/pki/root/sign-intermediate` endpoint for that functionality.)
 
-### Parameters
+**This is a potentially dangerous endpoint and only highly trusted users should
+have access.**
 
-- `name` `(string: <required>)` – Specifies the name of the role to create the
-  certificate against. This is part of the request URL.
+| Method | Path                                            | Issuer    |
+| :----- | :---------------------------------------------- | :-------- |
+| `POST` | `/pki/sign-verbatim(/:name)`                    | `default` |
+| `POST` | `/pki/issuer/:issuer_ref/sign-verbatim(/:name)` | Selected  |
```

Comment:
s/Selected/as provided/  or maybe :issuer_ref?  Not sure what's clean language here.